pynamodb/attributes.py

```diff
@@ -257,3 +261,223 @@ def deserialize(self, value):
         Takes a UTC datetime string and returns a datetime object
         """
         return parse(value, dayfirst=False).datetime
+
+
+class MapAttributeMeta(type):
+    def __init__(cls, name, bases, attrs):
```

Comment:
actually I'm a bit confused why this metaclass is necessary at all, can't we just set `_attributes` directly on MapAttribute as it's done so right now?